_visual/typography.md

```diff
@@ -175,8 +176,8 @@ Something about load performance consideration and how we recommend one of the f
             <p class="usa-monospace">
               font-family: ‘Merriweather’<br>
               font-weight: 400<br>
-              font-size: 22px<br>
-              line-height: 1.375em/30px
+              font-size: 20px<br>
+              line-height: 1.7em/34px
```

Comment:
@maya let's add it to this PR. There was also some lead paragraphs that I didn't know where to change the sizes and font families for. The things I missed are in the very first comment.